ipv-api/src/main/java/uk/gov/di/authentication/ipv/lambda/IPVCallbackHandler.java

```diff
@@ -148,7 +150,10 @@ public IPVCallbackHandler(ConfigurationService configurationService) {
         this.noSessionOrchestrationService =
                 new NoSessionOrchestrationService(configurationService);
         this.ipvCallbackHelper = new IPVCallbackHelper(configurationService);
-        this.authFrontend = new AuthFrontend(configurationService);
+        this.frontend =
+                configurationService.getOrchFrontendEnabled()
+                        ? new OrchFrontend(configurationService)
+                        : new AuthFrontend(configurationService);
     }
```

Comment:
Is it possible to write a test covering this?

Reply:
Not really possible to unit test the constructor given how it creates new instances of all the services it uses, but I can add a public method getFrontend(ConfigurationService) and call that from the constructor.

Reply:
Done. Might make sense to factor that out into a "FrontendProviderService" class if we're going to duplicate this logic elsewhere, but I think this makes sense for now.